backend/tests/conftest.py

```diff
@@ -48,6 +48,18 @@ def get_session_override():
     app.dependency_overrides.clear()
 
 
+@pytest.fixture
+def logged_in_client_2(memory_session: Session, auth_token_2: str):
```

Comment:
Wow, this took me a while to get :D How about `logged_in_client_user_2` or something like that?